Meridian59/Data/Models/AdminData/AdminObjectProperty.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.ComponentModel;
+
+namespace Meridian59.Data.Models.AdminData
+{
+    public class AdminObjectProperty : INotifyPropertyChanged
+    {
+        public string PropertyName { get; set; }
+        public string PropertyType { get; set; }
+        
+        private string _propertyValue { get; set; }
+        public string PropertyValue {
+            get { return _propertyValue; }
+            set
+            {
+                _propertyValue = value;
+                RaisePropertyChanged(new PropertyChangedEventArgs(_owner.ObjectNumber.ToString()));
```

Comment:
The argument of the 'PropertyChangedEventArgs' must be the name of the Property which changed its value.

In this case it would be:
>> RaisePropertyChanged(new PropertyChangedEventArgs("PropertyValue"));

I usually define all the property names as constants. The only other way to do this would be very slow 'reflection' calls.